### PR TITLE
testing

### PR DESCRIPTION
**Pull request by ndushay**
* testing as its own directory, with a couple links in the README

### CODE REVIEW

**Comment by ndushay** [comment]
D'oh - didn't see Jessie's earlier #1.   I'm now subscribed to this repo so I'll get emails.  D'oh.

**Comment by cbeer** [comment]
Do you want to remove the style section in favor of @jkeck 's?

**Comment by ndushay** [comment]
@cbeer yes.  done.  not bothering to rename branch, tho.

File: testing/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Continuous Integration
+
+* whenever possible, CI results should be visible with the code repo.
+
+### Travis
+
+* use travis for CI for github repos
+* put travis badge on repo README
+* (which rubies to test)
```
Comment (jkeck) [inline testing/README.md]: Should we mention anything about Jenkins on this page (perhaps in the context of internal services not available outside of the Stanford IP range)?

Comment (ndushay) [inline testing/README.md]: Good idea.  I just wanted to start the conversation.

Comment (jkeck) [inline testing/README.md]: :+1:

**Comment by jkeck** [comment]
:+1: let's link this to the top-level README or maybe https://github.com/sul-dlss/DeveloperPlaybook/blob/master/best-practices

**Comment by ndushay** [comment]
yeah, i wasn't sure if this should get its own top level thing or be under something else.  "best-practices" and "code-review" and "style"  seemed like pretty diff animals.  

What do you think of these top levels?
* coding practices  (includes style, documentation (incl gem version badge), versioning, code complexity, exception handling, robustness)
* licensing
* security
* deployment practices
* testing practices (includes CI, code coverage, dependency checking, ...)

**Comment by jkeck** [comment]
My only concern is that there may be some cross pollination between the first few categories.  However; I can't offer a better alternative so I'm fine with it unless anybody else has a better approach.

**Comment by mejackreed** [comment]
I'm hoping an organic organization emerges as more things get added.

**Comment by azaroth42** [comment]
Dead

**Pull request closed without merging**